CHARTER.md

```diff
@@ -0,0 +1,73 @@
+# vis.gl Charter
+
+Vis.gl aims to provide an open-source JavaScript framework for high-performance visualizations of large datasets on the web. It prioritizes strong support for geospatial data visualization, while also being flexible to cater to diverse non-geospatial visualization needs. It is designed to make advanced visualization techniques accessible to a wide range of developers by providing a user-friendly API and well-documented framework.
+
+## Section 0: Guiding Principles
+
+The `vis.gl` project is part of the [OpenJS Foundation](https://openjsf.org), which operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+
+## Section 1: Scope
+
+`vis.gl` is to provide a comprehensive, open-source solution for GPU-powered visualizations of large-scale datasets, in particular geospatial data visualizations. The `vis.gl` project consists of many different sub-projects, such as a layer-based visualization framework, GPU toolkit, basemap react components, and more. The project is designed to be modular, so that it can be used in a variety of different environments and for different purposes.
+
+
+### 1.1: In-scope
+
+ - Overseeing the **technical development** of the core vis.gl libraries and all affiliated sub-projects.
+ - **Release planning** and coordination
+ - Managing project repositories and **governance**, including issue tracking, pull request processes, and repository organization.
+ - Defining and enforcing **coding standards, testing practices, and development processes** to maintain code quality.
+ - Maintaining the list of **Maintainers and core contributors**, and setting permissions in project repositories.
+ - Reviewing and proposing **new project incubations or sub-projects**, as well as deciding on deprecations or significant changes to the project’s scope.
+ - Formation of technical **working groups**.
+ - **Representation** in technical collaborations with external communities.
```

Comment:
```suggestion
 Section Intentionally Left Blank
```

This is about the project's scope, not the TSC's responsibilities, so "problem spaces, use cases, repositories or other projects which are included with the work but may not be readily apparent."